scripts/travis/osx/build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+set -xe
+
+brew tap Homebrew/bundle
+brew bundle --file=scripts/travis/osx/Brewfile
+
+# build and test
+scons -Q --enable-werror --with-3rdparty=all test
```

Comment:
--with-3rdparty=openfec,cpputest test

Reply:
https://github.com/roc-project/roc/wiki/Building-%28native%29#macos

Reply:
Also, I think it makes sense to build both debug and release variants. And probably enable sanitizers in debug variant.

Reply:
Fixed